recipes/getdist/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "getdist" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/cmbant/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 8828031446fc91c0725f970c11bc79ad3442f8ea0360daebadd60ceaaed79a60
+
+build:
+  noarch: python
+  entry_points:
+    - getdist=getdist.command_line:getdist_command
+    - getdist-gui=getdist.command_line:getdist_gui
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - numpy >=1.17.0
+    - matplotlib-base (>=2.2.0,!=3.5.0)
+    - scipy >=1.5.0
+    - PyYAML >=5.1
+
+test:
+  commands:
+    - getdist --version
+
+about:
+  home: https://github.com/cmbant/getdist
+  summary: 'GetDist is a package for analysing Monte Carlo samples.'
+  description: |
+    GetDist is a Python package for analysing Monte Carlo samples, including 
+    correlated samples from Markov Chain Monte Carlo (MCMC).
+  license: LGPL-3.0-or-later
+  license_file: LICENSE.txt
+  doc_url: https://getdist.readthedocs.io/
+
+extra:
+  recipe-maintainers:
+    - vitenti
+    - marcpaterno
```

Comment:
All maintainers on the recipe besides the person who submitted need to comment here that they are ok maintaining the recipe.

Reply:
I am OK maintaining this recipe.